src/client/public/scripts/pages/repo.ts

```diff
@@ -0,0 +1,86 @@
+import '../components/filter-legend.js';
+
+import {html} from '../../../../../node_modules/lit-html/lib/lit-extended.js';
+import {TemplateResult} from '../../../../../node_modules/lit-html/lit-html.js';
+import * as api from '../../../../types/api.js';
+import {BaseElement, property} from '../components/base-element.js';
+import {FilterLegendEvent, FilterLegendItem} from '../components/filter-legend.js';
+import {filterController, FilterId} from '../dash/filter-controller.js';
+import {genericIssueListTemplate} from '../dash/issues.js';
+
+class RepoPage extends BaseElement {
+  @property({attribute: true}) path = '';
+  @property() untriaged: api.Issue[] = [];
+  private cachedId = '';
+  private filters: {[key: string]: [FilterLegendItem]} = {
```

Comment:
Does this need a filter given it's a single element?

Reply:
Gonna leave this in. I think I might factor out a generic concept of a filtered list.